backend/sqlite/options.go

```diff
@@ -9,10 +9,21 @@ type options struct {
 
 	// ApplyMigrations automatically applies database migrations on startup.
 	ApplyMigrations bool
+
+	// WorkerName allows setting a custom worker name. If not set, a random UUID will be generated.
+	WorkerName string
```

Comment:
Let's move this to the backend.Options. Some backends will not use this, but it's common enough that it makes sense to move it there. 

If we're logging the worker name in other backends, also make use of that option.